command/_errors.ts

```diff
@@ -1,7 +1,7 @@
 import { didYouMeanCommand } from "./_utils.ts";
 import type { Command } from "./command.ts";
 import { getFlag } from "../flags/_utils.ts";
```

Comment:
There are still some cross-package imports that need to be fixed, but these are mainly some helper functions that I don't really want to export.